### DISCUSSION TITLE
existing components using kebab case, jsx doesn't seem to support

### DISCUSSION THREAD

**Discussion by shi-yan** [solidjs/solid#1987] [Q&A]
Hello,

I want to develop photoshop plugins using solidjs. The plugin sdk comes with a set of ui components using the kebab case, for example, <sp-button> (https://developer.adobe.com/photoshop/uxp/2022/uxp-api/reference-spectrum/)

solidjs' jsx syntax seems to consider this as invalid. how can I workaround this issue?

Thanks!

**Comment by lxsmnsyc** [comment]
These are valid, but I'm curious what the issue is specifically.
